app/src/ui/release-notes/release-notes-dialog.tsx

```diff
@@ -187,4 +192,9 @@ export class ReleaseNotes extends React.Component<IReleaseNotesProps, {}> {
   private updateNow = () => {
     updateStore.quitAndInstallUpdate()
   }
+
+  private showAllReleaseNotes = () => {
+    const releaseNotesUri = 'https://desktop.github.com/release-notes/'
```

Comment:
This constant is already used [here](https://github.com/desktop/desktop/blob/master/app/src/ui/about/about.tsx#L46) and [here](https://github.com/desktop/desktop/blob/master/app/src/ui/banners/update-available.tsx#L51), and I'm adding another use now. Is there a mechanism through which they can be centralized so we don't repeat it?

Reply:
For things that are shared in multiple places, I'd create a module under `app/src/ui/lib/` to centralize it. If it were shared between the main and renderer processes I'd move it under `app/src/models` but for now this is only specific to the UI itself.

ere's a rough patch to illustrate the changes:

```diff
diff --git a/app/src/ui/about/about.tsx b/app/src/ui/about/about.tsx
index c690c4282c..dd94db4d3c 100644
--- a/app/src/ui/about/about.tsx
+++ b/app/src/ui/about/about.tsx
@@ -12,6 +12,7 @@ import { Disposable } from 'event-kit'
 import { Loading } from '../lib/loading'
 import { RelativeTime } from '../relative-time'
 import { assertNever } from '../../lib/fatal-error'
+import { ReleaseNotesUri } from '../lib/releases'
 
 interface IAboutProps {
   /**
@@ -42,9 +43,6 @@ interface IAboutProps {
 interface IAboutState {
   readonly updateState: IUpdateState
 }
-
-const releaseNotesUri = 'https://desktop.github.com/release-notes/'
-
 /**
  * A dialog that presents information about the
  * running application such as name and version.
@@ -246,7 +244,7 @@ export class About extends React.Component<IAboutProps, IAboutState> {
     const name = this.props.applicationName
     const version = this.props.applicationVersion
     const releaseNotesLink = (
-      <LinkButton uri={releaseNotesUri}>release notes</LinkButton>
+      <LinkButton uri={ReleaseNotesUri}>release notes</LinkButton>
     )
 
     return (
diff --git a/app/src/ui/banners/update-available.tsx b/app/src/ui/banners/update-available.tsx
index 2d734a799b..4001e3717d 100644
--- a/app/src/ui/banners/update-available.tsx
+++ b/app/src/ui/banners/update-available.tsx
@@ -8,6 +8,7 @@ import { shell } from '../../lib/app-shell'
 
 import { ReleaseSummary } from '../../models/release-notes'
 import { Banner } from './banner'
+import { ReleaseNotesUri } from '../lib/releases'
 
 interface IUpdateAvailableProps {
   readonly dispatcher: Dispatcher
@@ -48,8 +49,7 @@ export class UpdateAvailable extends React.Component<
     if (this.props.newRelease == null) {
       // if, for some reason we're not able to render the release notes we
       // should redirect the user to the website so we do _something_
-      const releaseNotesUri = 'https://desktop.github.com/release-notes/'
-      shell.openExternal(releaseNotesUri)
+      shell.openExternal(ReleaseNotesUri)
     } else {
       this.props.dispatcher.showPopup({
         type: PopupType.ReleaseNotes,
diff --git a/app/src/ui/lib/releases.ts b/app/src/ui/lib/releases.ts
new file mode 100644
index 0000000000..c9eb3a5b3b
--- /dev/null
+++ b/app/src/ui/lib/releases.ts
@@ -0,0 +1 @@
+export const ReleaseNotesUri = 'https://desktop.github.com/release-notes/'
```

Reply:
Thanks for the pointer. Added the constant.